reconcile/gitlab_housekeeping.py

```diff
@@ -159,25 +159,31 @@ def merge_merge_requests(dry_run, gl, merge_limit):
         if last_pipeline_result != 'success':
             continue
 
-        logging.info(['merge', gl.project.name, mr.iid])
         if not dry_run and merges < merge_limit:
+            logging.info(['merge', gl.project.name, mr.iid])
             mr.merge()
             merges += 1
 
 
-def run(dry_run=False, days_interval=15,
-        enable_closing=False, limit=1):
+def run(dry_run=False):
+    default_days_interval = 15
+    default_limit = 8
+    default_enable_closing = False
     instance = queries.get_gitlab_instance()
     settings = queries.get_app_interface_settings()
     repos = queries.get_repos_gitlab_housekeeping(server=instance['url'])
 
     for repo in repos:
+        hk = repo['housekeeping']
         project_url = repo['url']
+        days_interval = hk.get('days_interval') or default_days_interval
```

Comment:
what should happen when `days_interval` is `0`? here it is falling-back to `default_days_interval`.

Reply:
i think this is the desired behavior.

just in case, i'm not allowing 0 in the schema validation:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/5556/diffs?commit_id=7610d2006302e81ed61617ad521fcb02075128af